WooCommerce/Classes/ViewRelated/Products/Edit Product/Downloadable Files/File List/ProductDownloadListViewController.swift

```diff
@@ -39,7 +40,7 @@ final class ProductDownloadListViewController: UIViewController {
     private let localFileUploader: LocalFileUploader
 
     private var onDeviceMediaLibraryPickerCompletion: DeviceMediaLibraryPicker.Completion?
-    private var onWPMediaPickerCompletion: WordPressMediaLibraryImagePickerViewController.Completion?
+    private var onWPMediaPickerCompletion: WordPressMediaLibraryPickerViewController.Completion?
```

Comment:
> I find this confusing since we do have two different types Media and WordPressMedia, and it seems this difference should only be clear in the networking layer.

Exactly, that's why we should not interpret this property as returning type `WordPressMedia`, but as returning media from the WP library. I've renamed the property to `onWPLibraryPickerCompletion` to clarify the intention in 831cc34ed11085b5a04eea5953bb11ab525dcb2b.

I'd prefer keeping the completion type of `WordPressMediaLibraryPickerViewController` though. It might not be as readable as keeping the raw type, but it gives the context about where the closure is returned from. Does that make sense?